js/links.js

```diff
@@ -19,7 +20,20 @@ function controlLinks() {
 }
 
 const theLinks = controlLinks();
-const currentDiv = document.getElementById('links');
+const currentDiv = document.getElementById('linksOnPress');
+const parentNode = document.getElementById('parentLinksNode');
 
-window.onload = theLinks.removeLinksOnLoad();
-currentDiv.onclick = theLinks.addRemovedLinks;
\ No newline at end of file
+theLinks.removeLinksOnLoad();
+
+if (parentNode.hasChildNodes) {
+    console.log(parentNode.childElementCount);
+}
+
+currentDiv.onclick = function() {
```

Comment:
One thing we'll all want to be on the same page about is toggling the display of Links, Info & Todo. You've got this function here whereas I'm adding and removing a class with `display: none`. Later on down the road, it seems like we should be able to have one function that toggles each of those sections.